spyderlib/plugins/runconfig.py

```diff
@@ -62,8 +62,10 @@ def __init__(self, fname=None):
             self.wdir_enabled = True
         
     def set(self, options):
-        self.args = options.get('args', '')
-        self.args_enabled = options.get('args/enabled', False)
+        self.args = options.get('args',
+            CONF.get('run', 'args', ''))

```

Comment:
Why did you decide to use `CONF` in this PR? What was wrong with the previous approach?